src/main/kotlin/com/terraformation/backend/accelerator/db/ModuleStore.kt

```diff
@@ -0,0 +1,114 @@
+package com.terraformation.backend.accelerator.db
```

Comment:
Looking at the rest of the package, I believe it is better to move all of modules/events stuff out of the accelerator package, since it is focused on consoles.

Perhaps we can create a `com.terraformation.backend.participant` package, for the project-side of accelerator services.